amqp-quickstart/pom.xml

```diff
@@ -8,7 +8,7 @@
   <properties>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <surefire-plugin.version>2.22.0</surefire-plugin.version>
-    <quarkus.version>999-SNAPSHOT</quarkus.version>
+    <quarkus.version>0.18.0</quarkus.version>
```

Comment:
This shouldn't be done. `development` needs to stay to `999-SNAPSHOT`. Please revert this part.

Reply:
Sorry, I originally branched off `master`. Will revert it.